src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form 
+        class="form-search" 
+        action="/" 
+        method="get"
+      >
+        <input 
+          type="search" 
+          class="input-search input-search-big" 
+          name="search" 
+          placeholder="Try &quot;Los Angeles&quot;" 
+          data-qa="keypress big"
+        >
+      </form>
+      <form 
+        class="form-search" 
+        action="/" 
+        method="get"
+      >
+        <input 
+          type="search" 
+          class="input-search input-search-small" 
+          name="text" 
```

Comment:
```suggestion
          name="smallSearch" 
```
This name would be more correct